controllers/upgrade_controller.go

```diff
@@ -59,6 +59,8 @@ const plannedRequeueInterval = time.Minute * 2
 // UpgradeStateAnnotation is kept for backwards cleanup TODO: drop in 2 releases
 const UpgradeStateAnnotation = "nvidia.com/ofed-upgrade-state"
 
+const UpgradeSkipDrainLabelSelector = "nvidia.com/ofed-driver-upgrade-drain.skip!=true"
```

Comment:
a few nits.

1. mind putting all of these consts inside a const block ? 
2. add comment about this const what it means

Reply:
Thanks for the feedback! Addressed it